frontend/src/pages/TodoList/TodoList.tsx

```diff
@@ -10,6 +12,21 @@ const TodoList: FC = () => {
   const filtValue = useAppSelector((state) => state.todos.filtValue);
   const searchedValue = useAppSelector((state) => state.todos.searchValue);
 
+  const dispatch = useAppDispatch();
+
+  const fetchTodos = () => {
+    axios
+      .get(`https://mern-todo-app-a66w.onrender.com/`)
```

Comment:
Shouldn't be in component
Need to create folder 'api' and describe requests there